This diff was applied to libftgl sources at system build time, and is taken from
the MXE project: https://github.com/mxe/mxe/blob/c2970cf38e16d76b290108c21f266f8f7c37bb37/src/ftgl-2-obsolete-am-header.patch

MXE license:

Copyright © 2007–2014

    Volker Grabsch
    Mark Brand
    Tony Theodore
    Martin Gerhardy
    Tiancheng "Timothy" Gu
    ... and many other contributors

(contact via the project mailing list)

Permission is hereby granted, free of charge, to any person obtaining a copy of 
this software and associated documentation files (the "Software"), to deal in 
the Software without restriction, including without limitation the rights to 
use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of 
the Software, and to permit persons to whom the Software is furnished to do so, 
subject to the following conditions:

The above copyright notice and this permission notice shall be included in all 
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR 
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS 
FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR 
COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER 
IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN 
CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

diff -ur ftgl-2.1.3~rc5.orig/configure.ac ftgl-2.1.3~rc5/configure.ac
--- ftgl-2.1.3~rc5.orig/configure.ac	2013-04-25 16:57:09.000000000 +1000
+++ ftgl-2.1.3~rc5/configure.ac	2013-04-25 16:57:29.000000000 +1000
@@ -6,7 +6,7 @@
 AC_CONFIG_SRCDIR(src/FTPoint.cpp)
 AC_CONFIG_AUX_DIR(.auto)
 AM_INIT_AUTOMAKE([1.6 no-define dist-zip dist-bzip2])
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 FTGL_PROG_CXX
 
